src/Jenkins.ts

```diff
@@ -88,7 +88,12 @@ export class Jenkins {
       let result: JenkinsStatus;
       
       request
-        .get(url + '/api/json')
+        .get(url + '/api/json', {
+          'auth': {
```

Comment:
Forget this. It works also on my side (idk why it didn't work the first time). it should be ok on every version I guess. But I've you tried it on the  v1.647 but secured (it could have been some changes in the api between v1 and v2) ?